pylint_import_requirements/__init__.py

```diff
@@ -117,13 +128,21 @@ def visit_importfrom(self, node):
         names = [name for name, _alias in node.names]
         self.check_import(node, modname, names)
 
+    def open(self):
+        self.visited_distributions = set()
```

Comment:
I set it in `open()` as sort of symmetry with `close()`. I'm not sure if there is any reason to do this. In theory you could have multiple independent runs of this plugin with the same object. In that case the contract would be to call `open()` before and `close()` after every run. In our case this would not matter as the setup code happens in the constructor anyways